DataFormats/L1TCorrelator/interface/TkHTMiss.h

```diff
@@ -0,0 +1,62 @@
+#ifndef TkTrigger_TkHTMiss_h
+#define TkTrigger_TkHTMiss_h
+// Package:     L1Trigger
+// Class  :     TkHTMiss
+// Original Author:  E. Perez
+//         Created:  Nov 14, 2013
+
+// system include files
+// user include files
+#include "DataFormats/L1Trigger/interface/L1Candidate.h"
+#include "DataFormats/Common/interface/Ref.h"
+#include "DataFormats/Common/interface/RefProd.h"
+#include "DataFormats/Common/interface/Ref.h"
+#include "DataFormats/L1TCorrelator/interface/TkPrimaryVertex.h"
+#include "DataFormats/L1TCorrelator/interface/TkJet.h"
+#include "DataFormats/L1TCorrelator/interface/TkJetFwd.h"
+
+namespace l1t {
+  class TkHTMiss : public L1Candidate {
+  public:
+    TkHTMiss();
+    TkHTMiss(const LorentzVector& p4,
+             const double& EtTotal,
```

Comment:
```suggestion
             double EtTotal,
```
primitive types are more efficient passed by value

Reply:
done.